ivy/functional/backends/torch/experimental/linear_algebra.py

```diff
@@ -177,3 +178,27 @@ def cond(
 
 
 cond.support_native_out = False
+
+
+@with_supported_dtypes({"1.13.0 and below": ("float32", "float64", "complex64", "complex128")}, backend_version)
+def solve_triangular(
+    a: torch.Tensor,
+    b: torch.Tensor,
+    /,
+    *,
+    upper: bool = False,
+    left: bool = False,
+    unitriangular: bool = False,
```

Comment:
These are not the arguments you are returning from the `Ivy API`. Please make sure that the arguments are consistent across backends.

Reply:
For example, where are the arguments `lower`, `unit_diagonal` and `transpose`.